resources/fixtures/Showtec/Showtec-Compact-Power-Lightset-COB.qxf

```diff
@@ -17,7 +17,7 @@
   <Capability Min="30" Max="39" Preset="ColorMacro" Res1="#00ff00">Green</Capability>
   <Capability Min="40" Max="49" Preset="ColorMacro" Res1="#00ffff">Cyan</Capability>
   <Capability Min="50" Max="59" Preset="ColorMacro" Res1="#0000ff">Blue</Capability>
-  <Capability Min="60" Max="69" Preset="ColorMacro" Res1="#ff00ff">Purple</Capability>
+  <Capability Min="60" Max="69" Preset="ColorMacro" Res1="#a020f0">Purple</Capability>
```

Comment:
Why not Magenta here?

Reply:
Agree, makes more sense here by the patterns of the other colors.
Still the manual says Purple.
https://manualzz.com/doc/30363178/showtec-compact-power-lightset-v2-user-manual